Documentation/planning/arcade-powered-source-build/README.md

```diff
@@ -0,0 +1,170 @@
+# Arcade-powered source-build
+
+Source-build existing as a process outside the Microsoft build impedes efforts
+to maintain the source-buildability of .NET Core. Integration into the normal
+build process is driven by two major goals:
+
+1. When the .NET Core SDK **official build** completes, its artifacts include
+   validated, ship-ready source-build outputs, in addition to the Microsoft
+   build outputs.
+
+   * **Official builds** produce the signed bits released by Microsoft, running
+     on a daily or per-commit basis depending on repo and point in time.
+
+   * This has two core benefits over the current situation: we know immediately
+     whether an SDK can be source-built cleanly, and the status is as visible as
+     an SDK build failure.
+
+   * Breaks in source-build can be detected and fixed immediately by repo
+     owners, just as any other build failure, rather than fixed by source-build
+     maintainers weeks to months after the fact using patches.
+
+   * Decisions about rebuilding the SDK close to release day don't need to
+     consider source-build status and respin time as external factors.
```

Comment:
> and respin time as external factors.

I'm not sure what this is saying?

Reply:
Before this plan, when the Microsoft build respins, we have to get in gear in source-build to match it. How much bandwidth we have for this and how soon we can get it done is a complicated question. People making the decision of whether a specific fix needs a respin so it can get out in the current release could in theory take the external factor of source-build effort into consideration and deny fixes that would otherwise get in.

After this plan, this is no longer an external factor since source-build is part of the respin and our team isn't specifically needed to shepherd things along.

In practice, I don't think source-build overhead has much clout in this decision now, but if we wanted to get more serious about source-build sustainability without this plan, we would probably need the Microsoft build deciders to start considering this.

I'll see if I can describe this better.

Reply:
Changed to:

>   * Microsoft build process owners don't need to reject late build respins
     based on manual post-build source-build efforts being projected to not
     complete in time for release dates.

(Still not sure whether or not rejects would truly happen, but the core of that point is we (source-build) won't ever need to ask for that to be considered.)